Runtime/BacktraceClient.cs

```diff
@@ -598,6 +621,178 @@ public void Refresh()
                 AttributeProvider.AddDynamicAttributeProvider(_nativeClient);
             }
         }
+#if UNITY_WEBGL
+        private void InitializeWebGLOfflineSupport()
```

Comment:
It should be abstracted in the BacktraceDatabase implementation - we have the Start/Awake support there.